package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "messenger",
+  "version": "1.0.0",
+  "license": "MIT",
+  "browserslist": "> 0.5%, last 2 versions, not dead",
+  "scripts": {
+    "dev": "parcel src/index.hbs --no-cache",
+    "build": "parcel build src/index.hbs --no-cache",
+    "start": "parcel build src/index.hbs --no-cache && node app.js",
```

Можно переиспользовать
```suggestion
    "start": "npm run build && node app.js",
```

---

ok